frontend/src/k8sTypes.ts

```diff
@@ -1253,7 +1254,8 @@ export type HardwareProfileKind = K8sResourceCommon & {
     enabled: boolean;
     description?: string;
     tolerations?: Toleration[];
-    identifiers?: Identifier[];
+    identifiers?: (Identifier & { warning?: boolean })[];
+    warning?: WarningNotification;
```

Comment:
we do not want to add this warning as a part of the spec. we should just handle it ourselves and not store it. nothing is controlling this outside the UI so it is unreliable